src/scanners/javascript.js

```diff
@@ -34,11 +33,10 @@ export default class JavaScriptScanner {
       // ESLint is synchronous and doesn't accept streams, so we need to
       // pass it the entire source file as a string.
       var eslint = _ESLint.linter;
-      var rules = ignorePrivateFunctions(_rules);
 
-      for (let name in rules) {
+      for (let name in _rules) {
```

Comment:
As an aside: this `let`should probably be `const` see http://exploringjs.com/es6/ch_for-of.html#_iteration-variables-const-declarations-versus-var-declarations for the nuances.